static/reverse-context.js

```diff
@@ -20,6 +20,15 @@ window.__karma__ = (function(hasParent) {
 
 	function callParentKarmaMethod(methodName, args) {
 		args.unshift('iframe-test-results', methodName);
+		for (var i = 0, l = args.length; i < l; ++i) {
+			if (args[i] instanceof Error) {
+				args[i] = {
+					name: args[i].name,
```

Comment:
Maybe add a special property `'@@_serializedErrorFromIFrame': true` (or however obscure you want to name it), this way the check on the receiving end can be simplified to `if(typeof arg === 'object' && '@@_serializedErrorFromIFrame' in arg)`…